datm/atm_comp_nuopc.F90

```diff
@@ -424,7 +395,8 @@ subroutine InitializeRealize(gcomp, importState, exportState, clock, rc)
        call shr_sys_abort(subname//'Need to set attribute ScalarFieldIdxNextSwCday')
     endif
 
-    nextsw_cday = getNextRadCDay( current_ymd, current_tod, stepno, idt, iradsw, sdat%model_calendar )
+    nextsw_cday = getNextRadCDay(dayofYear, current_tod, stepno, idt, iradsw)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
```

Comment:
This check at line 399 is invalid/unnecessary since there is no rc argument in the preceeding calls.